common/src/fakeData/fakeDepartments.ts

```diff
@@ -1,44 +1,50 @@
+import faker from "faker";
 import { Department } from "../api/generated";
 
-export default (): Department[] => [
-  {
-    id: "dd9b0a72-e2b8-4e61-b7fb-79ebae7978fb",
-    departmentNumber: 4,
-    name: {
-      en: "Public Service Commission",
-      fr: "Commission de la fonction publique",
+export default (): Department[] => {
+  faker.seed(0); // repeatable results
+  faker.setLocale("en");
+
+  return [
+    {
+      id: faker.datatype.uuid(),
+      departmentNumber: faker.datatype.number(100),
```

Comment:
Is there a way to ensure each number is unique? Maybe a function that returns an incrementing number?

Reply:
I found that faker does have a function that guarantees uniqueness.  I applied to the department numbers in my latest commit to demonstrate.  I also removed the upper limit on random number generation.

Strangely, when I started applying it to the uuids it started throwing "failed to generate a unique" value pretty quickly.  It should be practically impossible to generate a collision on uuids so I'm not sure what's happening there.  🤷‍♂️

Reply:
Huh, that's odd, but yeah we don't need to use it for uuids. The whole point of uuids is that a collision will never happen in practice.